wasm/solana-client/Cargo.toml

```diff
@@ -11,14 +11,15 @@ keywords = ["solana", "wasm", "client", "rpc", "playground"]
 readme = "README.md"
 
 [features]
+cli = ["clap"]
 pubsub = ["js-sys", "web-sys", "wasm-bindgen", "wasm-bindgen-futures"]
 
 [dependencies]
 async-trait = "0.1"
 base64 = "0.13"
 bincode = "1.3"
 bs58 = "0.4"
-clap = "3.1.18"
+clap = { version = "4.5.4" , optional = true  }
```

Comment:
I think this upgrade will very likely cause all sorts of build errors in other WASM CLI crates we have. It would be better to not include this change with the PR.